x/staking/keeper/validator.go

```diff
@@ -53,6 +53,28 @@ func (k Keeper) mustGetValidatorByConsAddr(ctx sdk.Context, consAddr sdk.ConsAdd
 	return validator
 }
 
+func (k Keeper) GetValidatorByUnbondingOpId(
```

Comment:
There's a question about whether code is organized by feature or by data structure. So far, I have attempted to put my changes near the other code dealing with the same data structures, but I agree that it is probably not a great idea to add to this huge file.

I was thinking of breaking the entire file up into separate files for UBDs REDs, etc, but that would have to come after (or before) a PR to add our functionality to the SDK, otherwise it would be impossible to review code changes with all the whitespace changes.